### PR TITLE
Implemented CinemaHallController, OrderController,ShoppingCartControl…

### PR DESCRIPTION
**Pull request by vladkuzya**
…ler, UserController

### CODE REVIEW

**Review by margarita-ignatieva — APPROVED** [review]

**Comment by margarita-ignatieva** [comment]
looks good to me

File: src/main/java/mate/academy/controllers/CinemaHallController.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import mate.academy.service.mapper.CinemaHallMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private CinemaHallMapper cinemaHallMapper;
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/controllers/CinemaHallController.java]:
```suggestion
    private final CinemaHallMapper cinemaHallMapper;
```

File: src/main/java/mate/academy/model/dto/request/OrderRequestDto.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package mate.academy.model.dto.request;
+
+import java.util.List;
+import mate.academy.model.dto.response.TicketResponseDto;
+
+public class OrderRequestDto {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/model/dto/request/OrderRequestDto.java]: do you use this?

File: src/main/java/mate/academy/model/dto/response/TicketResponseDto.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.model.dto.response;
+
+import java.time.LocalDateTime;
+
+public class TicketResponseDto {
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/model/dto/response/TicketResponseDto.java]: same

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/model/dto/request/OrderRequestDto.java (hunk 2 above)
Comment (vladkuzya) [reply src/main/java/mate/academy/model/dto/request/OrderRequestDto.java]: No, i don't

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/java/mate/academy/controllers/CinemaHallController.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+import mate.academy.model.dto.request.CinemaHallRequestDto;
+import mate.academy.model.dto.response.CinemaHallResponseDto;
+import mate.academy.service.CinemaHallService;
+import mate.academy.service.mapper.CinemaHallMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController("/cinema-halls")
```
Comment (KseniiaMakarova) [inline src/main/java/mate/academy/controllers/CinemaHallController.java]: does it work? please attach the screenshot from Postman/browser where you fetch all cinema-halls

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/mate/academy/controllers/CinemaHallController.java (hunk 4 above)
Comment (vladkuzya) [reply src/main/java/mate/academy/controllers/CinemaHallController.java]: image
Теперь работает)

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**